vm/src/air_private_input.rs

```diff
@@ -215,7 +215,7 @@ mod tests {
         assert_matches::assert_matches,
     };
 
-    #[cfg(any(target_arch = "wasm32", no_std, not(feature = "std")))]
```

Comment:
why do we change this?

Reply:
It seems it is not a valid attribute for `#[cfg(...)]` in rust 1.80.0